changelogs/unreleased/-1wf8d38.yml

```diff
@@ -0,0 +1,6 @@
+---
+title: Implemented code to take instanceUrl from user and use it (#1wf8d38)
```

Comment:
Removed id from the title.

Reply:
Removed.
Thank You for reviewing my code.